<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QTableWidget" name="tableWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>4</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>Banane</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>Vert</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>Plus</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>38</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>Anana</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>Rouge</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>Moins</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>Faise</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>Jaune</string>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>Cl√©mentine</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>Bleu</string>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>+</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
